packages/meteor-otel/README.md

```
@@ -0,0 +1,256 @@
+});
+```
+
+## Features
+
+### 1. Automatic Instrumentation
+
+Out of the box, the package provides:
+- **Host metrics**: CPU, memory, network, disk
+- **Node.js runtime metrics**: Event loop, GC, heap usage
+
+> Note: Automatic instrumentation of core Node.js modules (such as `http`) and common drivers (MongoDB, etc.) is **not** yet provided out of the box by this package. For now, you must configure any desired instrumentations manually (see [Custom Instrumentations](#custom-instrumentations) below) and ensure you initialize OpenTelemetry **before** importing the modules you want instrumented.
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)